src/components/Search Component/Search.js

```diff
@@ -13,6 +18,10 @@ export default class Search extends React.Component {
   handleChange = (e) => {
     this.setState({input:e.target.value})
   }
+
+  alertUser = (message) => { 
+    alert(message);
```

Comment:
instead of an error, can we show the error below the search bar please

Reply:
Are there any other designs for it?

Reply:
Whatever you feel will make it easier. Go creative! ;)